pkgs/development/libraries/kquickimageedit/default.nix

```diff
@@ -0,0 +1,19 @@
+{ mkDerivation
+, fetchFromGitLab
+, extra-cmake-modules
+}:
+
+mkDerivation rec {
+  pname = "kquickimageeditor";
+  version = "0.1.2";
+
+  src = fetchFromGitLab {
+    domain = "invent.kde.org";
+    owner = "libraries";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-4X3GO/NPzA3c9KiGIkznLHUjLfNNNnPXif7IFwY5dOM=";
+  };
+
+  nativeBuildInputs = [ extra-cmake-modules ];
```

Comment:
Missing meta.

Reply:
We have historically not had `meta` info for the KDE framework libraries which is where this will be moving when it matures.